proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -574,6 +574,7 @@ message GenerateLastHistoryReplicationTasksRequest {
 }
 
 message GenerateLastHistoryReplicationTasksResponse {
+    int64 state_transition_count = 1;
```

Comment:
any upgrade/downgrade concern?

Reply:
old caller will get 0 here

and the system still use WaitN(ctx, 1) (before making the API call)
so for old logic, this will be a noop

for new logic, tokens consumed will be 1 per generate task call + n state transition from call result

Reply:
"old caller will get 0 here"
You mean new history client calls old history service?

Reply:
yes